src/wazuh_modules/vulnerability_scanner/src/policyManager/policyManager.hpp

```diff
@@ -150,6 +317,16 @@ class PolicyManager final : public Singleton<PolicyManager>
     {
         return m_configuration.at("indexer");
     }
+
+    /**
+     * @brief Get vulnerability detection configuration.
+     *
+     * @return nlohmann::json vulnerability-detection configuration.
+     */
+    nlohmann::json getVulnerabilityDetection() const
+    {
+        return m_configuration.at("vulnerability-detection");
```

Comment:
This may throw an exception and I don't see is being captured

Reply:
This function is called only if the key is present. The probability of throwing an exception is very low.
It is an approach similar to the previous one.